src/Components/Blinds/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+
+import './style.css';
+
+import blindsOpen from './blinds-open.svg';
+import blindsClosed from './blinds-closed.svg';
+
+const Blinds = (props) => {
+  const [isOpen, setIsOpen] = useState(props.blinds.state);
```

Comment:
když se podíváš co ti přijde (console.log), tak v props je `{blinds: {"open"}}`. Tj v `props.blinds.state` dostaneš undefined

Správně tedy máš mít 
`  const [isOpen, setIsOpen] = useState(props.blinds === "open");`
pokud to chceš mít jako true/false :)